testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -15,32 +15,33 @@ mkdir -p "$all_repos"
 all_venvs="$WORKSPACE/edx-venvs"
 mkdir -p "$all_venvs"
 
-for repo in "${REPOS[@]}"; do
+
+do_one_repo () {
   repo_dir="$all_repos/$repo"
```

Comment:
Should this be referencing the parameter you're passing the function instead of referencing `$repo`?

Reply:
Whoops, yes!

Reply:
I've rewritten that commit to fix that and make the function variables `local` for good measure.